web/Gruntfile.js

```diff
@@ -40,8 +45,12 @@ module.exports = function (grunt) {
                     '<%= config.srcScriptsDir %>/**/*.es.js'
                 ],
                 tasks: [
-                    'compilejs'
+                    'build'
                 ]
+            },
+            ngtemplates: {
+                files: ['app/templates/**/*.html'],
+                tasks: ['build']
```

Comment:
but this triggers a `template.js` compilation, which then needs to be compiled into `dist` files, because the `es` watcher only watches `es.js` source files.